workers/unity/Packages/com.improbable.gdk.legacy/Editor/BuildSystem/WorkerBuilder.cs

```diff
@@ -188,7 +189,11 @@ internal static IEnumerable<BuildTarget> GetUnityBuildTargets(SpatialBuildPlatfo
                     locationPathName = workerBuildData.BuildScratchDirectory
                 };
 
-                BuildPipeline.BuildPlayer(buildPlayerOptions);
+                var result = BuildPipeline.BuildPlayer(buildPlayerOptions);
+                if (result.summary.result != BuildResult.Succeeded)
+                {
+                    throw new BuildFailedException($"Build failed for {workerPlatform}");
```

Comment:
do we have info on why it failed and can add it to the exception?

Reply:
The Build itself prints all errors into the console.